modules/administration/nav-administration-guide.adoc

```diff
@@ -77,3 +77,4 @@ endif::[]
 *** xref:tshoot-saltboot.adoc[Troubleshooting Saltboot]
 *** xref:tshoot-sync.adoc[Troubleshooting Product Synchronization]
 *** xref:tshoot-taskomatic.adoc[Troubleshooting Taskomatic]
+*** xref:hostname-rename.adoc[Rename Server]
```

Comment:
If this is part of troubleshooting, you'll need to rename the file to be `tshoot-hostname-rename` and the chapter title to be "Troubleshooting Renaming the Hostname" (or similar). You'll also need to copy in the CCFR formatting advice from the other tshooting sections and edit the content accordingly.

Reply:
ok, but for the moment, I'll make it a chapter of its own.